apps/documents/views.py

```diff
@@ -44,7 +58,7 @@ def get_queryset(self):
 
     def perform_create(self, serializer):
         if settings.PROFILES_ENABLED:
-            created = serializer.save(owner_id=get_owner_id(self.request))
+            created = serializer.save(owner_id=get_owner_id(self.request), shipment_id=self.kwargs['shipment_pk'])
         else:
             created = serializer.save()
```

Comment:
Have you tested this with profiles_disabled? I feel like this might fail since the shipment_id isn't being populated in the .save()

Reply:
@jamesfneyer  good catch a specific test has been added in profiles disabled for both `documents` and `imports`.